src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+import { User } from './User';
+
+export interface Todo {
+  userId: number;
+  id: number;
+  title: string;
+  completed: boolean;
+}
+
+export interface TodoWithUsers extends Todo {
```

Comment:
it's only 1 user in one todo

```suggestion
export interface TodoWithUser extends Todo {
```